plugins/woocommerce/src/Database/Migrations/CustomOrderTable/CLIRunner.php

```diff
@@ -1213,6 +1226,104 @@ public function backfill( array $args = array(), array $assoc_args = array() ) {
+	 *
+	 * [--include-inactive]
+	 * : Include inactive plugins in the list.
+	 *
+	 * [--display-filenames]
+	 * : Print plugin file names instead of plugin names.
+	 *
+	 * @since 9.1.0
+	 *
+	 * @param array $args       Positional arguments passed to the command.
+	 * @param array $assoc_args Associative arguments (options) passed to the command.
+	 */
+	public function compatibility_info( array $args = array(), array $assoc_args = array() ) {
+		$container          = wc_get_container();
+		$feature_controller = $container->get( FeaturesController::class );
+		$plugin_info        = $feature_controller->get_compatible_plugins_for_feature( 'custom_order_tables', ! ( (bool) ( $assoc_args['include-inactive'] ?? null ) ) );
+		$display_filenames  = (bool) ( $assoc_args['display-filenames'] ?? null );
+
+		$compatibles       = $this->get_printable_plugin_names( $plugin_info['compatible'], $display_filenames );
+		$compatibles_count = count( $compatibles );
+		$this->log(
+			sprintf(
+				// translators: $1$d = plugins count, %2$s = colon (if list follows) or empty.
+				_n( "\n%%C%1\$d%%n compatible plugin found%2\$s", "\n%%C%1\$d%%n compatible plugins found%2\$s", $compatibles_count, 'woocommerce' ),
+				$compatibles_count,
+				$compatibles_count > 0 ? ":\n" : ''
+			)
+		);
+		$this->print_plugin_names( $compatibles );
+
+		$incompatibles       = $this->get_printable_plugin_names( $plugin_info['incompatible'], $display_filenames );
+		$incompatibles_count = count( $incompatibles );
+		$this->log(
+			sprintf(
+				// translators: $1$d = plugins count, %2$s = colon (if list follows) or empty.
+				_n( "\n%%C%1\$d%%n incompatible plugin found%2\$s", "\n%%C%1\$d%%n incompatible plugins found%2\$s", $incompatibles_count, 'woocommerce' ),
+				$incompatibles_count,
+				$incompatibles_count > 0 ? ":\n" : ''
+			)
+		);
+		$this->print_plugin_names( $incompatibles );
+
+		$uncertain       = $this->get_printable_plugin_names( $plugin_info['uncertain'], $display_filenames );
+		$uncertain_count = count( $uncertain );
+		$this->log(
+			sprintf(
+				// translators: $1$d = plugins count, %2$s = colon (if list follows) or empty.
+				_n( "\n%%C%1\$d%%n uncertain plugin found%2\$s", "\n%%C%1\$d%%n uncertain plugins found%2\$s", $uncertain_count, 'woocommerce' ),
+				$uncertain_count,
+				$uncertain_count > 0 ? ":\n" : ''
+			)
+		);
+		$this->print_plugin_names( $uncertain );
+	}
+
+	/**
+	 * Get the printable names for a set of plugins given their file names.
+	 *
+	 * @param array $plugins The plugin file names.
+	 * @param bool  $display_filenames True to simply return the sorted list of plugin file names.
+	 * @return array A sorted array of plugin names or file names.
+	 */
+	private function get_printable_plugin_names( array $plugins, bool $display_filenames ) {
```

Comment:
```suggestion
	private function get_printable_plugin_names( array $plugins, bool $display_filenames ): array {
```

(Similar to earlier comment, also optional.)